main-command/src/main/scala/sbt/BasicCommandStrings.scala

```diff
@@ -200,6 +200,12 @@ $AliasCommand name=
 	Starts the server if it has not been started. This is intended to be used with
 	-Dsbt.server.autostart=false."""
 
+  def Server: String = "server"
+  def ServerDetailed: String =
+    "--server always runs sbt in not-daemon mode."
+  def DashServer: String = "-server"
```

Comment:
I know there was something similar for client but if we are going to have two flags for server mode, I'd prefer `-s` and `--server`. Just dropping the single dash doesn't seem like much of a benefit. Not a big deal though.

Reply:
Yea. Single dash thing is a vestige from cargo culting bad `javac` and `scalac` options instead of following the Unix standards. I feel reluctant to add single letter option since we don't support composition of them (scopt does), so maybe we should just drop `-server` and keep only `--server`.

Reply:
Single dash dropped in https://github.com/sbt/sbt/pull/5823/commits/bba3d08aac9a3803e3d40e0d0e27c4f47b3541f1